modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -60,12 +62,12 @@ Before you implement multiple client providers in Anypoint Platform, carefully r
 * When you change the client provider for an existing API instance, ensure that the API:
 ** Has no contracts
 ** Has no client provider-related policies applied, such as PingFederate and OpenID policies
-* When you promote and import an API:
-
-* If previously did not assign a client provider to an environment, after you assign an external client provider, all existing APIs in that environment still use the native Anypoint Platform client provider. 
+* If previously did not assign a client provider to an environment when promoting and importing an API, after you assign an external client provider, all existing APIs in that environment still use the native Anypoint Platform client provider. 
```

Comment:
I did not understood this phrase, could we rephrase it?

I understand that the goal of the paragraph is to understand that, when you assign a client provider to an environment, all APIs created before such assignment, they will use the default provider. But a new API will get the external IdP assigned.

Reply:
Done! Thanks!